cmd/hope/utils/nodes.go

```diff
@@ -150,11 +155,23 @@ func GetMasters() (*[]hope.Node, error) {
 
 	for _, node := range *nodes {
```

Comment:
This could be optimized a bunch, but probably should be done as a whole push for caching hypervisor/node data, rather than starting here.